torchao/csrc/cuda/sparse_marlin/mem.h

```diff
@@ -19,6 +19,28 @@
 #include "base.h"
 
 namespace torchao {
+
```

Comment:
@xw285cornell looking for some quick advice, do you recommend we support AMD by adding conditional compilation flags to our existing cuda kernels or be OK with some more copy paste?

Reply:
Chatted offline and indeed ifdefs are the way to go